### ISSUE TITLE
UNABLE TO capture IMAGE

### ISSUE THREAD

**Issue by KingBrianZA** [jiangdongguo/AndroidUSBCamera#278]
when i click capture image it not saving it and the video is empty when i record

**Comment by quocnhat7** [comment]
perhaps your app did not request storage permission before staring the capturer code?
You can read the code again to find the reason

**Comment by burcutopcu** [comment]
I have the same issue, the video is empty when I record, did you solve it? @KingBrianZA

**Comment by vp-007** [comment]
I am facing the issue with video recording. 

However solved the image capture issue solved by changing following lines in AbstractUVCCameraHandler.java file:
```
case MSG_CAPTURE_STILL:
		  thread.handleCaptureStill((String) msg.obj);
//                thread.handleStillPicture((String) msg.obj);
                break;
```

**Comment by silvioalvim** [comment]
> I am facing the issue with video recording.
> 
> However solved the image capture issue solved by changing following lines in AbstractUVCCameraHandler.java file:
> 
> ```
> case MSG_CAPTURE_STILL:
> 		  thread.handleCaptureStill((String) msg.obj);
> //                thread.handleStillPicture((String) msg.obj);
>                 break;
> ```

Yes, thats fix the image capture, thank you! Any news about video recording?
